rules/ensure-system-master-group-is-not-set/rule.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "ensure-system-master-group-is-not-set",
+    "attributes": {
+      "armoBuiltin": true,
+      "actionRequired": "manual review",
+      "hostSensorRule": false,
+      "useFromKubescapeVersion": "",
+      "useUntilKubescapeVersion": "",
+      "imageScanRelated": false,
+      "controlConfigInputs": ""
```

Comment:
IMO no need to add empty attributes, it's clearer without

Reply:
Actually, I'd leave them. They could serves as placeholder for future rule improvements. WDYT?

Reply:
Hmm in what case do you imagine it would come in use? Pesonally I still think "less is more" here but let's talk about this if you want :) Maybe I'm not understanding the use case

Reply:
At first, it can help us standardizing the rules format.
Additionally, a contributor that wants to create his own rule can generate this with the scaffolder script. Having all the options available, the user can easily decide if one option can be useful or not for his rule. Otherwise he needs to know all the options available. In this way we automatically provide all the options that we support and the user can select the options that he needs and left the others unmodified. We can think to this as an easy way that we provide to the users to enrich the rules with built-in options that they may not know.

Reply:
I changed the attributes generation of `useUntilKubescapeVersion` and `useFromKubescapeVersion`, so now they are no more empty, unless the use give pass them as arguments.